internal/testrunner/runners/asset/runner.go

```diff
@@ -105,9 +105,7 @@ func (r *runner) run() ([]testrunner.TestResult, error) {
 		logger.Debug("removing package...")
 		err = packageInstaller.Uninstall()
 
-		// by default system package is part of an agent policy and it cannot be uninstalled
-		// https://github.com/elastic/elastic-package/blob/5f65dc29811c57454bc7142aaf73725b6d4dc8e6/internal/stack/_static/kibana.yml.tmpl#L62
```

Comment:
Elastic stacks with version 7.17.* still have `system` package installed (tested with 7.17.18).
![image](https://github.com/elastic/elastic-package/assets/5330827/d144959b-9855-4f2d-910d-e22459a0bf7a)

Would that be a problem ? @elastic/ecosystem 

Looking at the latest `system` package version , it requires Kibana version `^8.12.0`
https://github.com/elastic/integrations/blob/92900f6c0eb9cdc449808d8fd1dc234d05cf7f93/packages/system/manifest.yml#L11

Reply:
Ah, you are right, actually the part of the template modified in https://github.com/elastic/elastic-package/pull/1672/files#diff-df47a0840eb947003447c6e44e2fbec3f13b1ffda4478b46ba2cced8f4ae2fc6R51 is only used starting on 8.0.0.

In 7.x probably this policy was hard-coded in Kibana, so we will need to keep the exception at least for 7.x.

Reply:
Re-added this exception but just for Elastic stack `7.*` in this commit https://github.com/elastic/elastic-package/pull/1676/commits/7d4c3f5667da3f41b03b41c321ae68e4444fa993